lib/msal-common/src/cache/AccessTokenValue.ts

```diff
@@ -7,20 +7,30 @@
  * Value of access token cache item which is stored in cache. Helps manage token renewal.
  */
 export class AccessTokenValue {
+    // Success cases
+    tokenType?: string;
+    accessToken?: string;
+    idToken?: string;
+    refreshToken?: string;
+    expiresOnSec?: string;
+    extExpiresOnSec?: string;
 
-    tokenType: string;
-    accessToken: string;
-    idToken: string;
-    refreshToken: string;
-    expiresOnSec: string;
-    extExpiresOnSec: string;
+    // These fields exist when a thumbprint is being throttled
```

Comment:
Can we not have the same object for success and error for toke successful token response? can we have a common key but register token and errors as different Objects? I can sync up with you offline @jmckennon to discuss this.

Reply:
So I talked about this with Prithvi, and we can't have a common key because they will just overwrite each other, i.e. only an access token or only an error can be in the cache at any given time for a particular key, whereas we want to be able to have both. We'd need to key them differently--which also ends up taking up a lot of space. That's why I went with this implementation.